pom.xml

```diff
@@ -50,7 +50,7 @@
     <shade.prefix>${project.groupId}.shaded</shade.prefix>
     <spring-platform.version>2.0.8.RELEASE</spring-platform.version>
     <!-- BEGIN: AWS jdk version + dependencies -->
-    <aws-jdk.version>1.11.431</aws-jdk.version>
+    <aws-jdk.version>1.11.503</aws-jdk.version>
```

Comment:
They released another one 3 days back: 1.11.505. You might as well upgrade to that one.

https://github.com/aws/aws-sdk-java/releases